packages/web-console/src/scenes/Editor/Menu/index.tsx

```diff
@@ -278,6 +286,43 @@ const Menu = () => {
 
       <Separator />
 
+      <FeedbackDialog
+        withEmailInput
+        isSubmitting={isFeedbackSubmitting}
+        title="Web Console feedback"
+        subtitle="Let us know your thoughts"
+        trigger={({ setOpen }: { setOpen: (isOpen: boolean) => void }) => (
```

Comment:
```suggestion
        trigger={({ setOpen }) => (
```

should be inferred properly by typescript